src/Application/Interfaces/IAppUserRepository.cs

```diff
@@ -8,6 +8,11 @@
 namespace Application.Interfaces
 {
     public interface IAppUserRepository : IGenericRepository<AppUser>
-    {
+    {   
+        // for override
+        new Task<List<AppUser>> GetAll();
```

Comment:
why do we need a new keyword?

Reply:
if you want to override use override key

Reply:
Done